web-ui/src/components/member_selector/member_selector_dialog/MemberSelectorDialog.css

```diff
@@ -30,15 +30,15 @@
 }
 
 .member-selector-dialog .name-search-field {
-  width: 350px;
-  margin-right: 3rem;
```

Comment:
I think there was too much horizontal space between the "Name" input and the "Filter Type" dropdown, so I removed this `margin-right` property.